content/data/unicorns.json

```
@@ -520,5 +520,22 @@
 		"profileImg": "./markspratt.jpg",
 		"color": "#42AD8B",
 		"roles": ["author"]
+	},
+	{
+		"id": "ladybluenotes",
+		"name": "Sarah Gerrard",
+		"firstName": "Sarah",
+		"lastName": "Gerrard",
+		"description": "Full-Stack Software Developer and Technical Writter",
+		"socials": {
+			"twitter": "LadyBluenotes",
+			"github": "LadyBluenotes",
+			"linkedIn": "sarahgerrard",
+			"website": "https://saragerrard.me/"
```

```suggestion
			"website": "https://sarahgerrard.me/"
```

Hi Sarah - welcome! :)

Looks like this might be a typo, otherwise this looks good to merge!

---

@fennifith ahh! thank you so much for catching that

---

No problem! :D